.github/workflows/code-formatting.yml

```diff
@@ -0,0 +1,55 @@
+name: Code Formatting
+
+on:
+  pull_request:
+
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        ref: ${{ github.ref }}
+        fetch-depth: 0
+        submodules: true
```

Comment:
We do not have submodules. So you can remove this.

Reply:
Done !